tests/Doctrine/Tests/ORM/Hydration/AbstractHydratorTest.php

```diff
@@ -106,4 +107,40 @@ public function testHydrateAllRegistersAndClearsAllAttachedListeners(): void
 
         $this->hydrator->hydrateAll($this->mockStatement, $this->mockResultMapping);
     }
+
+    /**
+     * @group #DDC-8482
```

Comment:
`DDC-` is a prefix we used when we were using JIRA: https://github.com/doctrine/jira-github-issues/blob/89d31b0477cd0ade65535659c92f173f9ca83abc/projects.php#L4

You should probably use `GH` here instead, and that's if a group is really needed (not sure we recommend doing that in our documentation).